Crash when configuring Colors via Mod Menu

**Description of crash**
What were you doing when the game crashed?

I was using Mod Menu to customize the mod, I clicked on the Colors menu and then the game crashed.

**To Reproduce**
Steps to reproduce the crash:
1. Go to Mod Menu
2. Click on Coordinates Display
3. Click on the configure button
4. Click on the Colors button
5. Game crashes

**Version (please complete the following information):**
 - OS: Windows 11
 - Mod Loader: Fabric Loader 0.14.10
 - Mod Version: 2.1.3mc1.19.2
 - Minecraft Version: 1.19.2

**Error Log**
```
Prism Launcher version: 5.2


Launched instance in online mode

authserver.mojang.com resolves to:
    [108.138.89.47, 108.138.89.64, 108.138.89.60, 108.138.89.63]

session.minecraft.net resolves to:
    [35.153.153.200, 100.24.172.95, 34.197.32.8]

textures.minecraft.net resolves to:
    [13.226.225.60, 13.226.225.116, 13.226.225.49, 13.226.225.84]

api.mojang.com resolves to:
    [108.138.94.107, 108.138.94.20, 108.138.94.37, 108.138.94.91]


Minecraft folder is:
C:/Users/kiese/AppData/Roaming/PrismLauncher/instances/1.19.2 Optifine/.minecraft


Java path is:
C:/Program Files/Microsoft/jdk-17.0.5.8-hotspot/bin/javaw.exe


Checking Java version...
Java is version 17.0.5, using 64 (amd64) architecture, from Microsoft.


Main Class:
  net.fabricmc.loader.impl.launch.knot.KnotClient

Native path:
  C:/Users/kiese/AppData/Roaming/PrismLauncher/instances/1.19.2 Optifine/natives

Traits:
traits XR:Initial
traits FirstThreadOnMacOS

Libraries:
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/lwjgl/lwjgl-glfw-natives-windows-x86/3.3.1/lwjgl-glfw-natives-windows-x86-3.3.1.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/lwjgl/lwjgl-glfw-natives-windows/3.3.1/lwjgl-glfw-natives-windows-3.3.1.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/lwjgl/lwjgl-glfw/3.3.1/lwjgl-glfw-3.3.1.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/lwjgl/lwjgl-jemalloc-natives-windows-x86/3.3.1/lwjgl-jemalloc-natives-windows-x86-3.3.1.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/lwjgl/lwjgl-jemalloc-natives-windows/3.3.1/lwjgl-jemalloc-natives-windows-3.3.1.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/lwjgl/lwjgl-jemalloc/3.3.1/lwjgl-jemalloc-3.3.1.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/lwjgl/lwjgl-natives-windows-x86/3.3.1/lwjgl-natives-windows-x86-3.3.1.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/lwjgl/lwjgl-natives-windows/3.3.1/lwjgl-natives-windows-3.3.1.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/lwjgl/lwjgl-openal-natives-windows-x86/3.3.1/lwjgl-openal-natives-windows-x86-3.3.1.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/lwjgl/lwjgl-openal-natives-windows/3.3.1/lwjgl-openal-natives-windows-3.3.1.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/lwjgl/lwjgl-openal/3.3.1/lwjgl-openal-3.3.1.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/lwjgl/lwjgl-opengl-natives-windows-x86/3.3.1/lwjgl-opengl-natives-windows-x86-3.3.1.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/lwjgl/lwjgl-opengl-natives-windows/3.3.1/lwjgl-opengl-natives-windows-3.3.1.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/lwjgl/lwjgl-opengl/3.3.1/lwjgl-opengl-3.3.1.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/lwjgl/lwjgl-stb-natives-windows-x86/3.3.1/lwjgl-stb-natives-windows-x86-3.3.1.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/lwjgl/lwjgl-stb-natives-windows/3.3.1/lwjgl-stb-natives-windows-3.3.1.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/lwjgl/lwjgl-stb/3.3.1/lwjgl-stb-3.3.1.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/lwjgl/lwjgl-tinyfd-natives-windows-x86/3.3.1/lwjgl-tinyfd-natives-windows-x86-3.3.1.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/lwjgl/lwjgl-tinyfd-natives-windows/3.3.1/lwjgl-tinyfd-natives-windows-3.3.1.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/lwjgl/lwjgl-tinyfd/3.3.1/lwjgl-tinyfd-3.3.1.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/lwjgl/lwjgl/3.3.1/lwjgl-3.3.1.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/com/mojang/logging/1.0.0/logging-1.0.0.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/com/mojang/blocklist/1.0.10/blocklist-1.0.10.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/com/mojang/patchy/2.2.10/patchy-2.2.10.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/com/github/oshi/oshi-core/5.8.5/oshi-core-5.8.5.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/net/java/dev/jna/jna/5.10.0/jna-5.10.0.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/net/java/dev/jna/jna-platform/5.10.0/jna-platform-5.10.0.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/slf4j/slf4j-api/1.8.0-beta4/slf4j-api-1.8.0-beta4.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/apache/logging/log4j/log4j-slf4j18-impl/2.17.1/log4j-slf4j18-impl-2.17.1.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/com/ibm/icu/icu4j/70.1/icu4j-70.1.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/com/mojang/javabridge/1.2.24/javabridge-1.2.24.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/net/sf/jopt-simple/jopt-simple/5.0.4/jopt-simple-5.0.4.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/io/netty/netty-common/4.1.77.Final/netty-common-4.1.77.Final.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/io/netty/netty-buffer/4.1.77.Final/netty-buffer-4.1.77.Final.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/io/netty/netty-codec/4.1.77.Final/netty-codec-4.1.77.Final.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/io/netty/netty-handler/4.1.77.Final/netty-handler-4.1.77.Final.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/io/netty/netty-resolver/4.1.77.Final/netty-resolver-4.1.77.Final.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/io/netty/netty-transport/4.1.77.Final/netty-transport-4.1.77.Final.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/io/netty/netty-transport-native-unix-common/4.1.77.Final/netty-transport-native-unix-common-4.1.77.Final.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/io/netty/netty-transport-classes-epoll/4.1.77.Final/netty-transport-classes-epoll-4.1.77.Final.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/com/google/guava/failureaccess/1.0.1/failureaccess-1.0.1.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/com/google/guava/guava/31.0.1-jre/guava-31.0.1-jre.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/apache/commons/commons-lang3/3.12.0/commons-lang3-3.12.0.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/commons-io/commons-io/2.11.0/commons-io-2.11.0.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/commons-codec/commons-codec/1.15/commons-codec-1.15.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/com/mojang/brigadier/1.0.18/brigadier-1.0.18.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/com/mojang/datafixerupper/5.0.28/datafixerupper-5.0.28.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/com/google/code/gson/gson/2.8.9/gson-2.8.9.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/com/mojang/authlib/3.11.49/authlib-3.11.49.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/apache/commons/commons-compress/1.21/commons-compress-1.21.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/apache/httpcomponents/httpclient/4.5.13/httpclient-4.5.13.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/commons-logging/commons-logging/1.2/commons-logging-1.2.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/apache/httpcomponents/httpcore/4.4.14/httpcore-4.4.14.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/it/unimi/dsi/fastutil/8.5.6/fastutil-8.5.6.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/apache/logging/log4j/log4j-api/2.17.1/log4j-api-2.17.1.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/apache/logging/log4j/log4j-core/2.17.1/log4j-core-2.17.1.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/com/mojang/text2speech/1.13.9/text2speech-1.13.9.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/com/mojang/text2speech-natives-windows/1.13.9/text2speech-natives-windows-1.13.9.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/net/fabricmc/intermediary/1.19.2/intermediary-1.19.2.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/net/fabricmc/tiny-mappings-parser/0.3.0+build.17/tiny-mappings-parser-0.3.0+build.17.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/net/fabricmc/sponge-mixin/0.11.4+mixin.0.8.5/sponge-mixin-0.11.4+mixin.0.8.5.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/net/fabricmc/tiny-remapper/0.8.2/tiny-remapper-0.8.2.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/net/fabricmc/access-widener/2.1.0/access-widener-2.1.0.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/ow2/asm/asm/9.3/asm-9.3.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/ow2/asm/asm-analysis/9.3/asm-analysis-9.3.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/ow2/asm/asm-commons/9.3/asm-commons-9.3.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/ow2/asm/asm-tree/9.3/asm-tree-9.3.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/org/ow2/asm/asm-util/9.3/asm-util-9.3.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/net/fabricmc/fabric-loader/0.14.10/fabric-loader-0.14.10.jar
  C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/com/mojang/minecraft/1.19.2/minecraft-1.19.2-client.jar

Native libraries:

Mods:
  [✔] CoordinatesDisplay-2.1.3mc1.19.2
  [✔] fabric-api-0.67.0+1.19.2
  [✔] iris-mc1.19.2-1.4.2
  [✔] modmenu-4.1.1
  [✔] sodium-fabric-mc1.19.2-0.4.4+build.18

Params:
  --username  --version 1.19.2 --gameDir C:/Users/kiese/AppData/Roaming/PrismLauncher/instances/1.19.2 Optifine/.minecraft --assetsDir C:/Users/kiese/AppData/Roaming/PrismLauncher/assets --assetIndex 1.19 --uuid  --accessToken  --userType  --versionType release

Window size: 854 x 480

Java Arguments:
[-XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump, -Xms512m, -Xmx4096m, -Duser.language=en]


Minecraft process ID: 21012


[09:40:59] [main/INFO]: Loading Minecraft 1.19.2 with Fabric Loader 0.14.10
[09:40:59] [ForkJoinPool-1-worker-2/WARN]: Mod coordinatesdisplay uses the version 2.1.3mc1.19.2 which isn't compatible with Loader's extended semantic version format (Could not parse version number component '3mc1'!), SemVer is recommended for reliably evaluating dependencies and prioritizing newer version
[09:40:59] [ForkJoinPool-1-worker-2/WARN]: The mod "cotton-logging" contains invalid entries in its mod json:
- Unsupported root entry "licence" at line 7 column 12
[09:40:59] [ForkJoinPool-1-worker-1/WARN]: The mod "cotton-config" contains invalid entries in its mod json:
- Unsupported root entry "licence" at line 7 column 12
[09:40:59] [main/INFO]: Loading 64 mods:
	- coordinatesdisplay 2.1.3mc1.19.2
	- cotton-config 1.0.0-rc.7 via coordinatesdisplay
	- cotton-logging 1.0.0-rc.4 via coordinatesdisplay
	- fabric-api 0.67.0+1.19.2
	- fabric-api-base 0.4.12+93d8cb8290 via fabric-api
	- fabric-api-lookup-api-v1 1.6.11+93d8cb8290 via fabric-api
	- fabric-biome-api-v1 9.0.19+3dd584b490 via fabric-api
	- fabric-block-api-v1 1.0.0+12bfe4ea90 via fabric-api
	- fabric-blockrenderlayer-v1 1.1.21+c6af733c90 via fabric-api
	- fabric-client-tags-api-v1 1.0.2+b35fea8390 via fabric-api
	- fabric-command-api-v1 1.2.12+f71b366f90 via fabric-api
	- fabric-command-api-v2 2.1.8+93d8cb8290 via fabric-api
	- fabric-commands-v0 0.2.29+df3654b390 via fabric-api
	- fabric-containers-v0 0.1.36+df3654b390 via fabric-api
	- fabric-content-registries-v0 3.4.2+6064db2a90 via fabric-api
	- fabric-convention-tags-v1 1.1.2+93d8cb8290 via fabric-api
	- fabric-crash-report-info-v1 0.2.6+aeb40ebe90 via fabric-api
	- fabric-data-generation-api-v1 5.3.0+ad31814d90 via fabric-api
	- fabric-dimensions-v1 2.1.32+0dd10df690 via fabric-api
	- fabric-entity-events-v1 1.5.1+9244241690 via fabric-api
	- fabric-events-interaction-v0 0.4.29+c6af733c90 via fabric-api
	- fabric-events-lifecycle-v0 0.2.30+df3654b390 via fabric-api
	- fabric-game-rule-api-v1 1.0.22+c6af733c90 via fabric-api
	- fabric-item-api-v1 1.6.0+fa140d5990 via fabric-api
	- fabric-item-groups-v0 0.3.34+9244241690 via fabric-api
	- fabric-key-binding-api-v1 1.0.22+aaaf9d3390 via fabric-api
	- fabric-keybindings-v0 0.2.20+df3654b390 via fabric-api
	- fabric-lifecycle-events-v1 2.2.1+9244241690 via fabric-api
	- fabric-loot-api-v2 1.1.8+83a8659290 via fabric-api
	- fabric-loot-tables-v1 1.1.11+9e7660c690 via fabric-api
	- fabric-message-api-v1 5.0.4+93d8cb8290 via fabric-api
	- fabric-mining-level-api-v1 2.1.19+33fbc73890 via fabric-api
	- fabric-models-v0 0.3.18+c6af733c90 via fabric-api
	- fabric-networking-api-v1 1.2.6+9244241690 via fabric-api
	- fabric-networking-v0 0.3.23+df3654b390 via fabric-api
	- fabric-object-builder-api-v1 4.1.1+06e0e25190 via fabric-api
	- fabric-particles-v1 1.0.11+79adfe0a90 via fabric-api
	- fabric-registry-sync-v0 0.9.27+9244241690 via fabric-api
	- fabric-renderer-api-v1 1.0.13+12bfe4ea90 via fabric-api
	- fabric-renderer-indigo 0.6.15+aeb40ebe90 via fabric-api
	- fabric-renderer-registries-v1 3.2.21+df3654b390 via fabric-api
	- fabric-rendering-data-attachment-v1 0.3.16+12bfe4ea90 via fabric-api
	- fabric-rendering-fluids-v1 3.0.8+c6af733c90 via fabric-api
	- fabric-rendering-v0 1.1.24+df3654b390 via fabric-api
	- fabric-rendering-v1 1.11.0+73145abb90 via fabric-api
	- fabric-resource-conditions-api-v1 2.1.0+aae9039d90 via fabric-api
	- fabric-resource-loader-v0 0.8.0+f9594cb690 via fabric-api
	- fabric-screen-api-v1 1.0.28+28270f5890 via fabric-api
	- fabric-screen-handler-api-v1 1.3.2+1cc24b1b90 via fabric-api
	- fabric-sound-api-v1 1.0.0+c4f28df590 via fabric-api
	- fabric-textures-v0 1.0.21+aeb40ebe90 via fabric-api
	- fabric-transfer-api-v1 2.1.2+cd10d4fc90 via fabric-api
	- fabric-transitive-access-wideners-v1 1.3.1+42d99c3290 via fabric-api
	- fabricloader 0.14.10
	- io_github_douira_glsl-transformer 1.0.0-pre21.2 via iris
	- iris 1.4.2
	- jankson 3.0.0+j1.2.0 via coordinatesdisplay
	- java 17
	- minecraft 1.19.2
	- modmenu 4.1.1
	- org_anarres_jcpp 1.4.14 via iris
	- org_antlr_antlr4-runtime 4.10.1 via iris
	- org_joml_joml 1.10.4 via sodium
	- sodium 0.4.4+build.18
[09:40:59] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=file:/C:/Users/kiese/AppData/Roaming/PrismLauncher/libraries/net/fabricmc/sponge-mixin/0.11.4+mixin.0.8.5/sponge-mixin-0.11.4+mixin.0.8.5.jar Service=Knot/Fabric Env=CLIENT
[09:40:59] [main/INFO]: Compatibility level set to JAVA_17
[09:41:00] [main/INFO]: Loaded configuration file for Sodium: 29 options available, 1 override(s) found
[09:41:00] [main/WARN]: Force-disabling mixin 'features.sky.MixinWorldRenderer' as rule 'mixin.features.sky' (added by mods [iris]) disables it and children
[09:41:01] [main/WARN]: Configuration conflict: there is more than one oshi.properties file on the classpath
[09:41:02] [main/WARN]: Configuration conflict: there is more than one oshi.architecture.properties file on the classpath
[09:41:06] [main/INFO]: Building optimized datafixer
[09:41:10] [Render thread/WARN]: Method overwrite conflict for method_22920 in sodium.mixins.json:features.buffer_builder.intrinsics.MixinBufferBuilder from mod sodium, previously written by net.coderbot.iris.mixin.vertices.block_rendering.MixinBufferBuilder_SeparateAo. Skipping method.
[09:41:10] [Render thread/INFO]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[09:41:11] [Render thread/INFO]: Setting user: Xaeroxe
[09:41:12] [Render thread/INFO]: [CoordinatesDisplay]: Checking mod version...
[09:41:12] [Render thread/INFO]: [CoordinatesDisplay]: 1.19.2
[09:41:12] [Render thread/INFO]: [CoordinatesDisplay]: Initializing coordinatesdisplay v2.1.3...
[09:41:12] [Render thread/INFO]: [CoordinatesDisplay]: Loaded all config
[09:41:12] [Render thread/INFO]: [CoordinatesDisplay]: Parsed all color prefixes
[09:41:12] [Render thread/INFO]: [Indigo] Different rendering plugin detected; not applying Indigo.
[09:41:12] [ForkJoinPool.commonPool-worker-1/WARN]: [Iris Update Check] Cached update file detected, using that!
[09:41:13] [Render thread/INFO]: Backend library: LWJGL version 3.3.1 SNAPSHOT
[09:41:15] [Render thread/INFO]: ARB_direct_state_access detected, enabling DSA.
[09:41:15] [Render thread/INFO]: Debug functionality is disabled.
[09:41:15] [Render thread/INFO]: Profile: EXTREME (+3 options changed by user)
[09:41:17] [Render thread/INFO]: Using shaderpack: ComplementaryShaders_v4.6.zip
[09:41:17] [Render thread/WARN]: Static binding violation: PRIVATE @Overwrite method method_23182 in sodium.mixins.json:features.item.MixinItemRenderer from mod sodium cannot reduce visibiliy of PUBLIC target method, visibility will be upgraded.
[09:41:18] [Render thread/INFO]: Hardware information:
[09:41:18] [Render thread/INFO]: CPU: 16x AMD Ryzen 7 3700X 8-Core Processor 
[09:41:18] [Render thread/INFO]: GPU: NVIDIA GeForce RTX 3090/PCIe/SSE2 (Supports OpenGL 3.2.0 NVIDIA 526.98)
[09:41:18] [Render thread/INFO]: OS: Windows 11 (10.0)
[09:41:19] [Render thread/INFO]: Narrator library for x64 successfully loaded
[09:41:19] [Render thread/INFO]: Reloading ResourceManager: Default, Fabric Mods (Coordinates Display, Fabric API, Fabric API Base, Fabric API Lookup API (v1), Fabric Biome API (v1), Fabric BlockRenderLayer Registration (v1), Fabric Client Tags, Fabric Command API (v1), Fabric Command API (v2), Fabric Commands (v0), Fabric Containers (v0), Fabric Content Registries (v0), Fabric Convention Tags, Fabric Crash Report Info (v1), Fabric Data Generation API (v1), Fabric Dimensions API (v1), Fabric Entity Events (v1), Fabric Events Interaction (v0), Fabric Events Lifecycle (v0), Fabric Game Rule API (v1), Fabric Item API (v1), Fabric Item Groups (v0), Fabric Key Binding API (v1), Fabric Key Bindings (v0), Fabric Lifecycle Events (v1), Fabric Loot API (v2), Fabric Loot Tables (v1), Fabric Message API (v1), Fabric Mining Level API (v1), Fabric Models (v0), Fabric Networking API (v1), Fabric Networking (v0), Fabric Object Builder API (v1), Fabric Particles (v1), Fabric Registry Sync (v0), Fabric Renderer API (v1), Fabric Renderer - Indigo, Fabric Renderer Registries (v1), Fabric Rendering Data Attachment (v1), Fabric Rendering Fluids (v1), Fabric Rendering (v0), Fabric Rendering (v1), Fabric Resource Conditions API (v1), Fabric Resource Loader (v0), Fabric Screen API (v1), Fabric Screen Handler API (v1), Fabric Sound API (v1), Fabric Textures (v0), Fabric Transfer API (v1), Fabric Transitive Access Wideners (v1), Fabric Loader, Iris, Jankson, Mod Menu, Sodium)
[09:41:19] [Render thread/INFO]: Creating pipeline for dimension OVERWORLD
[09:41:23] [Render thread/WARN]: Removing unused function CheckForColor and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:23] [Render thread/WARN]: Removing unused function smoothstep1 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:23] [Render thread/WARN]: Removing unused function min2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:23] [Render thread/WARN]: Removing unused function max2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:23] [Render thread/WARN]: Removing unused function max2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:23] [Render thread/WARN]: Removing unused function smoothstep1 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:24] [Render thread/WARN]: Removing unused function sqrt1inv and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:24] [Render thread/WARN]: Removing unused function max2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:24] [Render thread/WARN]: Removing unused function min2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:24] [Render thread/WARN]: Removing unused function smoothstep1 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:24] [Render thread/WARN]: Removing unused function smoothstep1 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:24] [Render thread/WARN]: Removing unused function min2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:24] [Render thread/WARN]: Removing unused function min2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:24] [Render thread/WARN]: Removing unused function min2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:24] [Render thread/WARN]: Removing unused function max2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:24] [Render thread/WARN]: Removing unused function min2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:24] [Render thread/WARN]: Removing unused function CheckForColor and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:24] [Render thread/WARN]: Removing unused function min2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:24] [Render thread/WARN]: Removing unused function smoothstep1 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:24] [Render thread/WARN]: [Patcher] gl_FragColor is not supported yet, please use gl_FragData! Assuming that the shaderpack author intended to use gl_FragData[0]...
[09:41:24] [Render thread/WARN]: Removing unused function sqrt1inv and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:24] [Render thread/WARN]: Removing unused function CheckForColor and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:24] [Render thread/WARN]: Removing unused function min2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:24] [Render thread/WARN]: Removing unused function smoothstep1 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:24] [Render thread/WARN]: Removing unused function SampleTAAFilteredShadow and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:24] [Render thread/WARN]: Removing unused function min2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:24] [Render thread/WARN]: Removing unused function CheckForColor and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:25] [Render thread/WARN]: Removing unused function CheckForColor and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:25] [Render thread/WARN]: Removing unused function smoothstep1 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:25] [Render thread/WARN]: Removing unused function sqrt1inv and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:25] [Render thread/WARN]: Removing unused function CheckForColor and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:25] [Render thread/WARN]: Removing unused function CheckForColor and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:25] [Render thread/WARN]: Removing unused function CheckForColor and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:25] [Render thread/WARN]: Removing unused function sqrt1inv and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:25] [Render thread/WARN]: Removing unused function smoothstep1 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:25] [Render thread/WARN]: Removing unused function max2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:25] [Render thread/WARN]: Removing unused function CheckForColor and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:25] [Render thread/WARN]: Removing unused function CheckForColor and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:25] [Render thread/WARN]: Removing unused function max2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:25] [Render thread/WARN]: Removing unused function WavingWater and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:25] [Render thread/WARN]: Removing unused function SampleFilteredShadow and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:25] [Render thread/WARN]: Removing unused function calcLilypadMove and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:25] [Render thread/WARN]: Removing unused function SampleTAAFilteredShadow and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:25] [Render thread/WARN]: Removing unused function sqrt1inv and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:25] [Render thread/WARN]: Removing unused function GetMetalCol and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:25] [Render thread/WARN]: Removing unused function CheckForColor and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:25] [Render thread/WARN]: Removing unused function GetMetalCol and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:25] [Render thread/WARN]: Removing unused function min2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:25] [Render thread/WARN]: Removing unused function smoothstep1 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:26] [Render thread/WARN]: Removing unused function smoothstep1 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:26] [Render thread/WARN]: Removing unused function GetMetalCol and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:26] [Render thread/WARN]: Removing unused function min2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:26] [Render thread/WARN]: Removing unused function max2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:26] [Render thread/WARN]: Removing unused function CheckForColor and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:26] [Render thread/WARN]: Removing unused function GetLuminance and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:26] [Render thread/WARN]: Removing unused function max2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:26] [Render thread/WARN]: Removing unused function sqrt1inv and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:26] [Render thread/WARN]: Removing unused function smoothstep1 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:26] [Render thread/WARN]: Removing unused function smoothstep1 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:26] [Render thread/WARN]: Removing unused function min2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:26] [Render thread/WARN]: Removing unused function DecodeNormal and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:26] [Render thread/WARN]: Removing unused function smoothstep1 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:26] [Render thread/WARN]: Removing unused function max2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:26] [Render thread/WARN]: Removing unused function sqrt1inv and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:26] [Render thread/WARN]: Removing unused function GetMetalCol and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:26] [Render thread/WARN]: Removing unused function sqrt1inv and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:26] [Render thread/WARN]: Removing unused function GetLuminance and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:26] [Render thread/WARN]: Removing unused function smoothstep1 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:26] [Render thread/WARN]: Removing unused function SampleTAAFilteredShadow and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:26] [Render thread/WARN]: Removing unused function smoothstep1 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:26] [Render thread/WARN]: Removing unused function min2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:26] [Render thread/WARN]: Removing unused function min2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:26] [Render thread/WARN]: Removing unused function sqrt1inv and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:26] [Render thread/WARN]: Removing unused function min2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:26] [Render thread/WARN]: Removing unused function DecodeNormal and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:27] [Render thread/WARN]: Removing unused function sqrt1inv and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:27] [Render thread/WARN]: Removing unused function DecodeNormal and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:27] [Render thread/WARN]: Removing unused function smoothstep1 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:27] [Render thread/WARN]: Removing unused function Bayer256 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:27] [Render thread/WARN]: Removing unused function min2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:27] [Render thread/WARN]: Removing unused function min2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:27] [Render thread/WARN]: Removing unused function CheckForColor and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:27] [Render thread/WARN]: Removing unused function max2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:27] [Render thread/WARN]: Removing unused function sqrt1inv and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:27] [Render thread/WARN]: Removing unused function smoothstep1 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:27] [Render thread/WARN]: Removing unused function sqrt1inv and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:27] [Render thread/WARN]: Removing unused function SampleFilteredShadow and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:27] [Render thread/WARN]: Removing unused function smoothstep1 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:27] [Render thread/WARN]: Removing unused function sqrt1inv and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:27] [Render thread/WARN]: Removing unused function max2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:27] [Render thread/WARN]: Removing unused function smoothstep1 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:27] [Render thread/WARN]: Removing unused function smoothstep1 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:27] [Render thread/WARN]: Removing unused function smoothstep1 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:27] [Render thread/WARN]: Removing unused function min2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:27] [Render thread/WARN]: Removing unused function sqrt1inv and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:27] [Render thread/WARN]: Removing unused function calcLilypadMove and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:27] [Render thread/WARN]: Removing unused function smoothstep1 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:27] [Render thread/WARN]: Removing unused function smoothstep1 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:27] [Render thread/WARN]: Removing unused function max2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:27] [Render thread/WARN]: Removing unused function smoothstep1 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:27] [Render thread/WARN]: Removing unused function max2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:27] [Render thread/WARN]: Removing unused function smoothstep1 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:27] [Render thread/WARN]: Removing unused function CheckForColor and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:27] [Render thread/WARN]: Removing unused function sqrt1inv and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:27] [Render thread/WARN]: Removing unused function smoothstep1 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:27] [Render thread/WARN]: Removing unused function max2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:28] [Render thread/WARN]: Removing unused function Bayer256 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:28] [Render thread/WARN]: Removing unused function WavingWater and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:28] [Render thread/WARN]: Removing unused function Bayer256 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:28] [Render thread/WARN]: Removing unused function smoothstep1 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:28] [Render thread/WARN]: Removing unused function max2 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:28] [Render thread/WARN]: Removing unused function WavingWater and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:28] [Render thread/WARN]: Removing unused function Bayer256 and omitting further such messages outside of debug mode. See debugging.md for more information.
[09:41:28] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.play
[09:41:28] [Render thread/WARN]: Missing sound for event: minecraft:entity.goat.screaming.horn_break
[09:41:29] [Render thread/INFO]: OpenAL initialized on device OpenAL Soft on Headphones (Bose PC Desktop Controller)
[09:41:29] [Render thread/INFO]: Sound engine started
[09:41:29] [Render thread/INFO]: Created: 1024x512x4 minecraft:textures/atlas/blocks.png-atlas
[09:41:29] [Render thread/INFO]: Created: 256x128x4 minecraft:textures/atlas/signs.png-atlas
[09:41:29] [Render thread/INFO]: Created: 512x512x4 minecraft:textures/atlas/banner_patterns.png-atlas
[09:41:29] [Render thread/INFO]: Created: 512x512x4 minecraft:textures/atlas/shield_patterns.png-atlas
[09:41:29] [Render thread/INFO]: Created: 256x256x4 minecraft:textures/atlas/chest.png-atlas
[09:41:29] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/beds.png-atlas
[09:41:29] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/shulker_boxes.png-atlas
[09:41:30] [Render thread/WARN]: Shader rendertype_entity_translucent_emissive could not find sampler named Sampler2 in the specified shader program.
[09:41:30] [Render thread/INFO]: Created: 256x256x0 minecraft:textures/atlas/particles.png-atlas
[09:41:30] [Render thread/INFO]: Created: 256x256x0 minecraft:textures/atlas/paintings.png-atlas
[09:41:30] [Render thread/INFO]: Created: 128x128x0 minecraft:textures/atlas/mob_effects.png-atlas
[09:41:31] [Worker-Bootstrap-9/INFO]: 4283 Datafixer optimizations took 24849 milliseconds
[09:42:45] [Render thread/INFO]: [CoordinatesDisplay]: Save pressed so saving config
[09:43:45] [Render thread/WARN]: Configuration conflict: there is more than one oshi.architecture.properties file on the classpath
[09:43:46] [Render thread/ERROR]: Reported exception thrown!
net.minecraft.class_148: Rendering screen
	at net.minecraft.class_757.method_3192(class_757.java:890) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1177) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:768) [client-intermediary.jar:?]
	at net.minecraft.client.main.Main.method_44604(Main.java:244) [client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:51) [client-intermediary.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461) [fabric-loader-0.14.10.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) [fabric-loader-0.14.10.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.14.10.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at org.prismlauncher.impl.OneSixLauncher.invokeMain(OneSixLauncher.java:104) [NewLaunch.jar:?]
	at org.prismlauncher.impl.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:176) [NewLaunch.jar:?]
	at org.prismlauncher.impl.OneSixLauncher.launch(OneSixLauncher.java:186) [NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:144) [NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:74) [NewLaunch.jar:?]
Caused by: java.lang.ArithmeticException: / by zero
	at me.boxadactle.coordinatesdisplay.gui.config.ColorConfigScreen.renderColorPicker(ColorConfigScreen.java:80) ~[CoordinatesDisplay-2.1.3mc1.19.2.jar:?]
	at me.boxadactle.coordinatesdisplay.gui.config.ColorConfigScreen.method_25394(ColorConfigScreen.java:110) ~[CoordinatesDisplay-2.1.3mc1.19.2.jar:?]
	at net.minecraft.class_757.method_3192(class_757.java:881) ~[client-intermediary.jar:?]
	... 16 more
---- Minecraft Crash Report ----
// Uh... Did I do that?
Time: 2022-11-20 09:43:46
Description: Rendering screen
java.lang.ArithmeticException: / by zero
	at me.boxadactle.coordinatesdisplay.gui.config.ColorConfigScreen.renderColorPicker(ColorConfigScreen.java:80)
	at me.boxadactle.coordinatesdisplay.gui.config.ColorConfigScreen.method_25394(ColorConfigScreen.java:110)
	at net.minecraft.class_757.method_3192(class_757.java:881)
	at net.minecraft.class_310.method_1523(class_310.java:1177)
	at net.minecraft.class_310.method_1514(class_310.java:768)
	at net.minecraft.client.main.Main.method_44604(Main.java:244)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.prismlauncher.impl.OneSixLauncher.invokeMain(OneSixLauncher.java:104)
	at org.prismlauncher.impl.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:176)
	at org.prismlauncher.impl.OneSixLauncher.launch(OneSixLauncher.java:186)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:144)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:74)
A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------
-- Head --
Thread: Render thread
Stacktrace:
	at me.boxadactle.coordinatesdisplay.gui.config.ColorConfigScreen.renderColorPicker(ColorConfigScreen.java:80)
	at me.boxadactle.coordinatesdisplay.gui.config.ColorConfigScreen.method_25394(ColorConfigScreen.java:110)
-- Screen render details --
Details:
	Screen name: me.boxadactle.coordinatesdisplay.gui.config.ColorConfigScreen
	Mouse location: Scaled: (160, 74). Absolute: (964.000000, 445.000000)
	Screen size: Scaled: (427, 240). Absolute: (2560, 1440). Scale factor of 6.000000
Stacktrace:
	at net.minecraft.class_757.method_3192(class_757.java:881)
	at net.minecraft.class_310.method_1523(class_310.java:1177)
	at net.minecraft.class_310.method_1514(class_310.java:768)
	at net.minecraft.client.main.Main.method_44604(Main.java:244)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.prismlauncher.impl.OneSixLauncher.invokeMain(OneSixLauncher.java:104)
	at org.prismlauncher.impl.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:176)
	at org.prismlauncher.impl.OneSixLauncher.launch(OneSixLauncher.java:186)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:144)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:74)
-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: Default, Fabric Mods
Stacktrace:
	at net.minecraft.class_6360.method_36565(class_6360.java:49)
	at net.minecraft.class_310.method_1587(class_310.java:2348)
	at net.minecraft.class_310.method_1514(class_310.java:787)
	at net.minecraft.client.main.Main.method_44604(Main.java:244)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.prismlauncher.impl.OneSixLauncher.invokeMain(OneSixLauncher.java:104)
	at org.prismlauncher.impl.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:176)
	at org.prismlauncher.impl.OneSixLauncher.launch(OneSixLauncher.java:186)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:144)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:74)
-- System Details --
Details:
	Minecraft Version: 1.19.2
	Minecraft Version ID: 1.19.2
	Operating System: Windows 11 (amd64) version 10.0
	Java Version: 17.0.5, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode, sharing), Microsoft
	Memory: 709477296 bytes (676 MiB) / 2717908992 bytes (2592 MiB) up to 4294967296 bytes (4096 MiB)
	CPUs: 16
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 7 3700X 8-Core Processor             
	Identifier: AuthenticAMD Family 23 Model 113 Stepping 0
	Microarchitecture: unknown
	Frequency (GHz): 3.59
	Number of physical packages: 1
	Number of physical CPUs: 8
	Number of logical CPUs: 16
	Graphics card #0 name: NVIDIA GeForce RTX 3090
	Graphics card #0 vendor: NVIDIA (0x10de)
	Graphics card #0 VRAM (MB): 4095.00
	Graphics card #0 deviceId: 0x2204
	Graphics card #0 versionInfo: DriverVersion=31.0.15.2698
	Memory slot #0 capacity (MB): 8192.00
	Memory slot #0 clockSpeed (GHz): 2.13
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 8192.00
	Memory slot #1 clockSpeed (GHz): 2.13
	Memory slot #1 type: DDR4
	Virtual memory max (MB): 21954.56
	Virtual memory used (MB): 17246.40
	Swap memory total (MB): 5661.47
	Swap memory used (MB): 97.12
	JVM Flags: 3 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms512m -Xmx4096m
	Fabric Mods: 
		coordinatesdisplay: Coordinates Display 2.1.3mc1.19.2
			cotton-config: Cotton Config 1.0.0-rc.7
			cotton-logging: Cotton Logging 1.0.0-rc.4
			jankson: Jankson 3.0.0+j1.2.0
		fabric-api: Fabric API 0.67.0+1.19.2
			fabric-api-base: Fabric API Base 0.4.12+93d8cb8290
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.11+93d8cb8290
			fabric-biome-api-v1: Fabric Biome API (v1) 9.0.19+3dd584b490
			fabric-block-api-v1: Fabric Block API (v1) 1.0.0+12bfe4ea90
			fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.21+c6af733c90
			fabric-client-tags-api-v1: Fabric Client Tags 1.0.2+b35fea8390
			fabric-command-api-v1: Fabric Command API (v1) 1.2.12+f71b366f90
			fabric-command-api-v2: Fabric Command API (v2) 2.1.8+93d8cb8290
			fabric-commands-v0: Fabric Commands (v0) 0.2.29+df3654b390
			fabric-containers-v0: Fabric Containers (v0) 0.1.36+df3654b390
			fabric-content-registries-v0: Fabric Content Registries (v0) 3.4.2+6064db2a90
			fabric-convention-tags-v1: Fabric Convention Tags 1.1.2+93d8cb8290
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.2.6+aeb40ebe90
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 5.3.0+ad31814d90
			fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.32+0dd10df690
			fabric-entity-events-v1: Fabric Entity Events (v1) 1.5.1+9244241690
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.29+c6af733c90
			fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.30+df3654b390
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.22+c6af733c90
			fabric-item-api-v1: Fabric Item API (v1) 1.6.0+fa140d5990
			fabric-item-groups-v0: Fabric Item Groups (v0) 0.3.34+9244241690
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.22+aaaf9d3390
			fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.20+df3654b390
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.2.1+9244241690
			fabric-loot-api-v2: Fabric Loot API (v2) 1.1.8+83a8659290
			fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.1.11+9e7660c690
			fabric-message-api-v1: Fabric Message API (v1) 5.0.4+93d8cb8290
			fabric-mining-level-api-v1: Fabric Mining Level API (v1) 2.1.19+33fbc73890
			fabric-models-v0: Fabric Models (v0) 0.3.18+c6af733c90
			fabric-networking-api-v1: Fabric Networking API (v1) 1.2.6+9244241690
			fabric-networking-v0: Fabric Networking (v0) 0.3.23+df3654b390
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 4.1.1+06e0e25190
			fabric-particles-v1: Fabric Particles (v1) 1.0.11+79adfe0a90
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.9.27+9244241690
			fabric-renderer-api-v1: Fabric Renderer API (v1) 1.0.13+12bfe4ea90
			fabric-renderer-indigo: Fabric Renderer - Indigo 0.6.15+aeb40ebe90
			fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.21+df3654b390
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.16+12bfe4ea90
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.0.8+c6af733c90
			fabric-rendering-v0: Fabric Rendering (v0) 1.1.24+df3654b390
			fabric-rendering-v1: Fabric Rendering (v1) 1.11.0+73145abb90
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 2.1.0+aae9039d90
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.8.0+f9594cb690
			fabric-screen-api-v1: Fabric Screen API (v1) 1.0.28+28270f5890
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.2+1cc24b1b90
			fabric-sound-api-v1: Fabric Sound API (v1) 1.0.0+c4f28df590
			fabric-textures-v0: Fabric Textures (v0) 1.0.21+aeb40ebe90
			fabric-transfer-api-v1: Fabric Transfer API (v1) 2.1.2+cd10d4fc90
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 1.3.1+42d99c3290
		fabricloader: Fabric Loader 0.14.10
		iris: Iris 1.4.2
			io_github_douira_glsl-transformer: glsl-transformer 1.0.0-pre21.2
			org_anarres_jcpp: jcpp 1.4.14
			org_antlr_antlr4-runtime: antlr4-runtime 4.10.1
		java: OpenJDK 64-Bit Server VM 17
		minecraft: Minecraft 1.19.2
		modmenu: Mod Menu 4.1.1
		sodium: Sodium 0.4.4+build.18
			org_joml_joml: joml 1.10.4
	Loaded Shaderpack: ComplementaryShaders_v4.6.zip
		Profile: EXTREME (+3 options changed by user)
	Launched Version: 1.19.2
	Backend library: LWJGL version 3.3.1 SNAPSHOT
	Backend API: NVIDIA GeForce RTX 3090/PCIe/SSE2 GL version 3.2.0 NVIDIA 526.98, NVIDIA Corporation
	Window size: 2560x1440
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	Graphics mode: fancy
	Resource Packs: Fabric Mods
	Current Language: English (US)
	CPU: 16x AMD Ryzen 7 3700X 8-Core Processor 
#@!@# Game crashed! Crash report saved to: #@!@# C:\Users\kiese\AppData\Roaming\PrismLauncher\instances\1.19.2 Optifine\.minecraft\crash-reports\crash-2022-11-20_09.43.46-client.txt
Process exited with code -1.
```

--- a/fabric/src/main/java/me/boxadactle/coordinatesdisplay/gui/config/ColorConfigScreen.java
+++ b/fabric/src/main/java/me/boxadactle/coordinatesdisplay/gui/config/ColorConfigScreen.java
@@ -77,8 +77,8 @@ public class ColorConfigScreen extends Screen {
         int b = (int) (this.height / s);
         int c = (int) (a / 2 + (5 / s));
         int d = (int) (b / 2.3) - 4;
-        int e = (int) (872 / 1.8) / this.client.options.getGuiScale().getValue();
-        int f = (int) (586 / 1.8) / this.client.options.getGuiScale().getValue();
+        int e = (int) (872 / 1.8) / this.nonZeroGuiScale();
+        int f = (int) (586 / 1.8) / this.nonZeroGuiScale();
 
         RenderSystem.setShader(GameRenderer::getPositionTexShader);
         RenderSystem.setShaderColor(1.0F, 1.0F, 1.0F, 1.0F);
@@ -87,6 +87,16 @@ public class ColorConfigScreen extends Screen {
         drawTexture(matrices, c, d, 0.0F, 0.0F, e, f, e, f);
 
         matrices.pop();
+    }
+
+    private int nonZeroGuiScale() {
+        int scale = this.client.options.getGuiScale().getValue();
+        if (scale == 0) {
+            // This formula copied from the Minecraft wiki
+            return (int) Math.max(1, Math.min(Math.floor(this.width / 320), Math.floor(this.height / 240)));
+        } else {
+            return scale;
+        }
     }
 
     @Override

